packages/Select/src/Select.js

```diff
@@ -1,94 +1,139 @@
 import React from "react";
-import classNames from "classnames";
 import PropTypes from "prop-types";
+import { callAll, extractChildrenProps } from "@paprika/helpers";
 import * as constants from "@paprika/constants/lib/Constants";
+import SelectPropsCollector from "./SelectPropsCollector";
 import * as sc from "./Select.styles";
 
 const Select = React.forwardRef((props, ref) => {
-  const renderPlaceholder = () => {
-    const { placeholder } = props;
-    if (!placeholder) return null;
-    return (
-      <option disabled value="" key="placeholder">
-        {placeholder}
-      </option>
-    );
-  };
-
   const {
     a11yText,
-    className,
     children,
+    defaultValue,
     hasError,
     isDisabled,
     isReadOnly,
-    size,
+    onChange,
     placeholder,
+    size,
     value,
     ...moreProps
   } = props;
 
-  if (a11yText || placeholder) {
-    moreProps["aria-label"] = a11yText || placeholder;
+  const isControlled = value !== undefined;
+  const containerProps = extractChildrenProps(children, SelectPropsCollector);
+  const [hasNoValue, setHasNoValue] = React.useState(isControlled ? !value : !defaultValue);
+
+  const _refSelect = React.useRef();
+  const refSelect = ref || _refSelect;
```

Comment:
why not just use `ref` ?

Reply:
I don't think it should be required for the consumer to pass a `ref` to the `<Select>`.  So I want it to work whether they do or not. I do reference the `refSelect` in the code, so there needs to be one either way.

Reply:
Where did you use `refSelect ` ? I only found a storybook example using it

Reply:
Oh! I did have a hook I was using it in, but I think I found a simpler way to do what I was trying to do (change the styling when the placeholder item is selected).  So you are right, I don't need to define my own, I can just use `ref` 👍